src/libs/runtime-utils.js

```diff
@@ -195,11 +195,12 @@ export const getGalaxyCost = (app, dataDiskSize) => {
  */
 export const getGalaxyComputeCost = app => {
   const appStatus = app?.status?.toUpperCase()
-  const defaultNodepoolComputeCost = machineCost('n1-standard-1')
+  const defaultNodepoolComputeCost = getHourlyCostForMachineType('n1-standard-1', defaultComputeRegion, false)
```

Comment:
I know it's not your PR but I wonder if we can opportunistically unhardcode `n1-standard-1` and instead define it with something like `defaultNodepoolMachineType`?

Reply:
It may also help to have a comment saying we use `defaultComputeRegion` because we're not yet enabling other locations for Galaxy apps.

Reply:
Done. Turns out theres already a constant named `defaultGceMachineType` that is equal to `n1-standard-1`